common/TelemetryEvents/SetupFlow/Environments/EnvironmentCreationEvent.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using DevHome.Telemetry;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+
+namespace DevHome.Common.TelemetryEvents.SetupFlow.Environments;
+
+public enum EnvironmentsTelemetryStatus
+{
+    Started,
+    Succeeded,
+    Failed,
```

Comment:
Does DevHome log why the status is failed?  I don't think the message needs to go to telemetry.  Just, somewhere developer can look.

Reply:
yes we log it here if there was an exception:
https://github.com/microsoft/devhome/blob/037291b4d2ee4f1b0b279eb06eab7b8a75d0970c/common/Environments/Models/CreateComputeSystemOperation.cs#L115

But I realized we don't check inside the CreateComputeSystemResult to log that failure message from the extension. So I can add that.

Reply:
added a log function to log the data instead the *Result when there is a failure